src/ceph-disk

```diff
@@ -1977,6 +2014,8 @@ def activate(
                 )
             if conf_val is not None:
                 init = conf_val
+            elif is_systemd():
+                init = 'systemd'
```

Comment:
I think this will switch to systemd init for all distros with this change, not just Ubuntu.  Are we ready for that?  I think currently a rhel7.x node would still get set to sysvinit.